Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -4122,41 +4140,69 @@ def prettify_applications_arr(applications_arr: Union[List[dict], dict]):
         applications_arr = [applications_arr]
     for i in range(len(applications_arr)):
         application = applications_arr[i]
+        application_characteristics_list=[]
+        for key,value in application.items():
```

Comment:
can we have more "meaningful" other than key, value? what's the key here and what's the value?